### DISCUSSION TITLE
kubernetes-hardening-guidance/introduction/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#79] [Announcements]
# kubernetes-hardening-guidance/introduction/

Kubernetes，经常被缩写为 “K8s”，是一个开源的容器或编排系统，用于自动部署、扩展和管理容器化应用程序。它管理着构成集群的所有元素，从应用中的每个微服务到整个集群。与

https://lib.jimmysong.io/kubernetes-hardening-guidance/introduction/

**Comment by wenyurush** [comment]
图裂了

**Reply by rootsongjc** [reply]
图片路径引用错误，已修复。

**Comment by wenyurush** [comment]
1.2版本出了，会更新么

**Reply by rootsongjc** [reply]
我暂时没有精力来更新。
